app/sidekiq/lighthouse/create_intent_to_file_job.rb

```diff
@@ -26,6 +26,7 @@ class InvalidITFTypeError < StandardError; end
     # exhausted attempts will be logged in intent_to_file_queue_exhaustions table
     sidekiq_options retry: 14, queue: 'low'
     sidekiq_retries_exhausted do |msg, error|
+      ::Rails.logger.info("Create Intent to File Job exhausted all retries for in_progress_form_id: #{msg['args'][0]}")
```

Comment:
Should this be using `log_silent_failure_avoided` from lib/zero_silent_failures/monitor.rb

Reply:
i can certainly add that in as well, but i wanted a unique message i can filter in the logs by as well.

Reply:
@wayne-weibel should i add the ZSF function to both exhaustions and regular failures?

Reply:
Only needed for the exhaustion